bindings/python/multibody/data.hpp

```diff
@@ -13,6 +13,7 @@
 #include <eigenpy/memory.hpp>
 #include <eigenpy/exception.hpp>
 
+#include "pinocchio/bindings/python/multibody/model.hpp" // Requested for memory alignment issues
```

Comment:
Probably you need to change the inclusion order. It seems to me that we need to include first `std-vector.hpp` and `std-aligned-vector.hpp`, right?

Reply:
I tried it locally, and it doesn't solve neither the issue in my PC.

Reply:
No, we don't care. They are not involved in your script.